index.d.ts

```diff
@@ -110,4 +111,6 @@ type PickerStateProviderProps = {
 
 export const PickerStateProvider: React.ComponentType<PickerStateProviderProps>;
 
-export const PickerAvoidingView: React.ComponentType<React.PropsWithChildren>;
+type PickerAvoidingViewProps = ViewProps & { enabled?: boolean; children?: ReactNode };
```

Comment:
I guess `ViewProps` already includes `children?: React.ReactNode`, wdyt?
```suggestion
type PickerAvoidingViewProps = ViewProps & { enabled?: boolean };
```